Uncertainty_Quantification/Deep500_recipe/recipe_UQ.py

```diff
@@ -0,0 +1,97 @@
+""" A recipe to reproduce the UQ of the following repo:
+    https://github.com/spcl/deep-weather
+
+"""
+import math
+
+import deep500 as d5
+# Using PyTorch as the framework
+import deep500.frameworks.pytorch as d5fw
+
+import torch
+from events import *
+from models import *
+from utils import *
+from losses import *
+import loader
+from scheduler import CyclicLRScheduler
+
+args = args_parser()
+
+
+def model(batch_size, in_channels, out_channels, classes=None, shape=None):
+    net = resnet2d_simple(
+        in_channels=in_channels,
+        out_channels=out_channels,
+    )
+    return d5fw.PyTorchNativeNetwork(net), "input", "output"
+
+
+def executor():
+    mod, _, _ = model(
+        None,
+        in_channels=len(args.parameters) * len(args.time_steps) * 2,
+        out_channels=1,
+        classes=None,
+        shape=None,
+    )
+    loss_op = MixLoss(torch.nn.L1Loss(), MSSSIMLoss(), 0.84)
+    return d5fw.PyTorchNativeGraphExecutor(mod.module, loss_op, "input")
+
+
+# Fixed Components
+FIXED = {
+    "model": model,
+    "model_kwargs": {
+        "in_channels": len(args.parameters) * len(args.time_steps) * 2,
+        "out_channels": 1,
+    },
+    "dataset": loader.callable_weather_dataset(args),
+    "epochs": args.epochs,
+}
+
+# Mutable Components
+MUTABLE = {
+    "batch_size":
+    args.batch_size,
+    "executor":
+    executor(),
```

Comment:
styling question - why are the values in a different line than the keys? I'd put them on the same line